src/App.js

```diff
@@ -24,6 +27,9 @@ const App = () => {
         <Route path="/dishes/:dishId" element={<RecipePage />} />
         <Route path="/generate_menu" element={<GenerateMenuPage />} />
         <Route path="/generated_menu" element={<GeneratedMenuPage />} />
+        <Route path="/my_cookbook/mycookbook" element={<MyCookbook />} />
+        <Route path="/my_cookbook/login" element={<LoginPage />} />
+        <Route path="/my_cookbook/register" element={<RegisterPage />} />

```

Comment:
Do we need to nest those routes in `my_cookbook`? Logging and registering in will affect whole site not just the page with our recipies.
Also is there any reason to write`my_cookbook` as parent route but then `mycookbook` without underscore as child route?